lib/proxy_api/external_ipam.rb

```diff
@@ -63,7 +63,7 @@ def initialize(args)
     #     {"error": "Unable to connect to External IPAM server"}
     def next_ip(subnet, mac, group = "")
       raise "subnet cannot be nil" if subnet.nil?
-      response = parse get("/subnet/#{subnet}/next_ip?mac=#{mac}&group=#{URI.escape(group.to_s)}")
+      response = parse get("/subnet/#{subnet}/next_ip?mac=#{mac}&group=#{CGI.escape(group.to_s)}")
```

Comment:
How about:
```suggestion
      response = parse get("/subnet/#{subnet}/next_ip?#{URI.encode_www_form(mac: mac, group: group)}")
```

Reply:
yes, you are right, thank you/
Do you think I should change the other references? they have only one param from what I saw.